docs/spec/v1beta2/kustomization.md

```diff
@@ -748,6 +748,16 @@ With `spec.postBuild.substituteFrom` you can provide a list of ConfigMaps and Se
 from which the variables are loaded.
 The ConfigMap and Secret data keys are used as the var names.
 
+The `spec.postBuild.substituteFrom.optional` field indicates how the
+controller should handle a referenced ConfigMap or Secret being absent
+at renconciliation time. The controller's default behavior―with
```

Comment:
```suggestion
at renconciliation time. The controller's default behavior ― with
```

Reply:
Ah, spaces around your em dashes, then? I can meet you there.